superset-frontend/src/components/DvtTextareaSelectRun/dvt-textarea-select-run.module.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { keyframes, styled } from '@superset-ui/core';
+
+const dropdownKeyframes = keyframes`
+  from {
+    transform: scaleY(0);
+  }
+  to {
+    transform: scaleY(1);
+  }
+`;
+
+interface StyledDvtTextareaIconProps {
+  isOpen: boolean;
+}
+
+interface StyledDvtTextareaItemProps {
+  selectedItem: number;
+  Item: number;
+}
+
+const StyledDvtTextareaSelectRun = styled.div`
+  position: relative;
+  width: 1126px;
+  height: 281px;
+  border-radius: 12px;
+  background: ${({ theme }) => theme.colors.grayscale.light5};
+`;
+
+const StyledDvtTextarea = styled.textarea`
+  width: 1066px;
+  height: 221px;
+  border: none;
+  color: ${({ theme }) => theme.colors.dvt.text.help};
+  resize: none;
+  margin: 30px 30px 0 30px;
+  &:focus {
+    outline: none;
+  }
+`;
+
+const StyledDvtTextareaLimit = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  cursor: pointer;
+`;
```

Comment:
StyledDvtTextareaLimit min-width vermelisin. StyledDvtTextareaDropdown max-width verdiğinde aynısı pixel olmalı dropdown sabit olması için

Reply:
ya da min-width: 105px veya 120px olabilir